sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -124,7 +124,7 @@ export interface CertificateOperation {
    */
   certificateTransparency?: boolean;
   /**
-   * The certificate signing request (CSR) that is being used in the certificate operation.
+   * The base64-encoded certificate signing request (CSR) that is being used in the certificate operation.
```

Comment:
But they pass in a `Uint8Array`. base64 encoding should be done by the implementation. Same with all instances below.

Reply:
Gotcha. This PR makes no sense then. I'll close it. Let me know if you have an idea of what to do here, besides "just making sure that it works". I do appreciate that you're helping us make sure things are correct!